core/src/main/java/ai/timefold/solver/core/impl/bavet/common/index/ContainIndexer.java

```diff
@@ -0,0 +1,67 @@
+package ai.timefold.solver.core.impl.bavet.common.index;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+
+import ai.timefold.solver.core.impl.util.ListEntry;
+
+import org.jspecify.annotations.NullMarked;
+
+@NullMarked
+final class ContainIndexer<T, Key_, CollectionKey extends Collection<Key_>> implements Indexer<T> {
```

Comment:
The PR description says that the indexer implementations are present. Yet I see `UnsupportedOperationException` everywhere. What am I missing?